seastar/build.rs

```diff
@@ -4,9 +4,11 @@ static CXX_BRIDGES: &[&str] = &[
     // Put all files that contain a cxx::bridge into this list
     "src/preempt.rs",
     "src/config_and_start_seastar.rs",
+    "src/api_safety.rs",
+    "src/spawn.rs",
 ];
 
-static CXX_CPP_SOURCES: &[&str] = &["src/config_and_start_seastar.cc"];
+static CXX_CPP_SOURCES: &[&str] = &["src/config_and_start_seastar.cc", "src/spawn.cc"];
```

Comment:
One more thing. As each PR will add files to this variable, it would be well to have each file put in a separate line so that we reduce the risk of a merge conflict.

I think a trick used in `CXX_BRIDGES` will prevent rustfmt from putting everything back into a single line: just add a comment similar to the `Put all files...`. Please add it.